express-app/models/comentario.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+//require document class from camo
+var EmbeddedDocument = require('camo').EmbeddedDocument;
+
+
+/*
+ * Comment of a Family regarding its economic status
+ * 
+ * Attributes:
+ * ---------------
+ * family : FR Family 
```

Comment:
What is this FR? is not in the model, you need in the model or is only a embedded document